Packages/com.unity.inputsystem/InputSystem/InputUpdateType.cs

```diff
@@ -71,6 +71,14 @@ public enum InputUpdateType
         /// </remarks>
         ManualThreaded = Manual | 1 << 5,
 
+        /// <summary>
+        /// Turning this flag on allowed for updates to run even when Unity is running in the background.
+        /// </summary>
+        /// <remarks>
+        /// This doesn't guarantee that all devices will update properly while in the background, and so you will still only recieve updates for devices that support running in the background natively.
+        /// </remarks>
+        IgnoreFocus = 1 << 31,
```

Comment:
This shouldn't be exposed. I think it's an okay-ish thing for the native runtime to roll it into the update mask, but `InputUpdateType` should only have proper update types.